recipes/perl-class-accessor/recipe.yaml

```
@@ -0,0 +1,48 @@
+schema_version: 1
+
+context:
+  name: perl-class-accessor
+  version: 0.51
+  sha256: bf12a3e5de5a2c6e8a447b364f4f5a050bf74624c56e315022ae7992ff2f411c
+
+package:
+  name: ${{ name }}
+  version: ${{ version }}
+
+source:
+  url: https://cpan.metacpan.org/authors/id/K/KA/KASEI/Class-Accessor-${{ version }}.tar.gz
+  sha256: ${{ sha256 }}
+
+build:
+  number: 0
+  noarch: generic
+  script:
+    - perl Makefile.PL INSTALLDIRS=site NO_PERLLOCAL=1 NO_PACKLIST=1
+    - make
+    - make test
+    - make install
```

```suggestion
  script: |
    perl Makefile.PL INSTALLDIRS=site NO_PERLLOCAL=1 NO_PACKLIST=1
    make
    make test
    make install
    cp ${PREFIX}/man/man1/perlartistic.1 .
    cp ${PREFIX}/man/man1/perlgpl.1 .
```
Let's this format for the script to improve readability.  The last two lines are only needed if we need the `perl_5` license - if the package has its own `LICENSE` file, we don't copy anything.